src/Tribe/Customizer/Single_Event.php

```diff
@@ -92,48 +94,67 @@ public function setup() {
 	 */
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
+		
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
+			$manager,
+			$customizer->get_setting_name( 'post_title_heading', $section ),
+			[
+				'label'   => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
 
 		$manager->add_setting(
 			$customizer->get_setting_name( 'post_title_color', $section ),
-			array(
+			[
 				'default'              => $this->get_default( 'post_title_color' ),
 				'type'                 => 'option',
-
 				'sanitize_callback'    => 'sanitize_hex_color',
 				'sanitize_js_callback' => 'maybe_hash_hex_color',
-			)
+			]
 		);
 
 		$manager->add_control(
 			new WP_Customize_Color_Control(
 				$manager,
 				$customizer->get_setting_name( 'post_title_color', $section ),
-				array(
+				[
 					'label'   => tribe_events_views_v2_is_enabled() ? esc_html__( 'Event Title Color', 'the-events-calendar' ) : esc_html__( 'Post Title Color', 'the-events-calendar' ),
 					'section' => $section->id,
-				)
+				]
 			)
 		);
+		
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
+			$manager,
+			$customizer->get_setting_name( 'details_bg_color_heading', $section ),
+			[
+				'label'   => esc_html__( 'Adjust Appearance', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: ad75d64a424c46efc5f4e30645aab3a6</sub>